src/containers/Profile/ProfileDetails.js

```diff
@@ -32,11 +74,40 @@ const ProfileDetailsComponent = props => {
     toggleNetworkSwitch,
     currentTab,
     setActiveTab,
-    setReviewsModalVisible
+    setReviewsModalVisible,
+    onEdit,
+    onCloseProgressBar,
+    showBanner
   } = props;
 
   let bodyClass;
 
+  const percentageComplete = getProfileCompletePercent(user);
+
+  const pageBanner = (
+    <React.Fragment>
+      <PageBanner>
+        <span style={{ width: '50px' }} />
+        <ProgressBar
+          heading="Profile Strength"
+          percentage={percentageComplete}
+        />
+        <span style={{ width: '20px' }} />
+        <span className="width-stretch">
+          <Button onClick={onEdit} type="link">
+            Edit Profile
+          </Button>
+          <span style={{ float: 'right' }}>
+            <Button onClick={onCloseProgressBar} type="link">
```

Comment:
The X should be built into the pagebanner component. And it should be there if the onClose prop is passed through to it.